.gitignore

```diff
@@ -3,7 +3,7 @@
 .idea/misc.xml
 .idea/uiDesigner.xml
 .idea/google-java-format.xml
-config
+config.txt
```

Comment:
you shouldn't keep config files in your repo directory even if you aren't checking it in. it's bad practice.